inst/schema/spec.md

```diff
@@ -135,21 +135,24 @@ Plotly documentation: https://plotly.com/javascript/plotly-fundamentals/
     },
     "data": [
          {
-             cols: ["prev_year_1", "prev_year_2", "prev_year_3"],
+             x_col: "month",
+             y_col: "value", 
              id: "none",
              name: "No intervention",
              type: "lines",
              marker: {color: "grey"}
          },
          {
-             cols: ["prev_year_1", "prev_year_2", "prev_year_3"],
+             x_col: "month",
+             y_col: "value", 
```

Comment:
Prototype supporting both this and the original wide format config (since that remains useful for the bar graph) here: https://github.com/mrc-ide/vue-prototypes/pull/5/files